src/modules/trayicon/libkvitrayicon.cpp

```diff
@@ -172,9 +164,7 @@ void KviTrayIconWidget::disableTrayIcon()
 	g_pMainWindow->executeInternalCommand(KVI_INTERNALCOMMAND_TRAYICON_HIDE);
 }
 
-#define NIDLEMSGS 18
-
-static const char * idlemsgs[NIDLEMSGS] = {
+static const char * idlemsgs[] = {
```

Comment:
these idle messages... does this even work? Only time I see these is in the code.